runtime/compiler/z/codegen/J9UnresolvedDataSnippet.hpp

```diff
@@ -73,6 +74,8 @@ class UnresolvedDataSnippet : public J9::UnresolvedDataSnippet
 
    public:
 
+   TR::Instruction* fenceNOPInst;
```

Comment:
We should put this under a setter and not expose the field publicly. Also add a getter.